components/aws/sagemaker/tests/unit_tests/requirements.txt

```diff
@@ -1,6 +1,6 @@
-boto3==1.12.33
+boto3==1.14.12
 coverage==5.1
 pathlib2==2.3.5
 pytest==5.4.1
 pyyaml==5.3.1
-sagemaker==1.56.1
\ No newline at end of file
+sagemaker==2.1.0
```

Comment:
@RedbackThomson can an improvement to reduce duplication be part of your PR ?

Dustin had to change sagemaker version at multiple places